src/Storages/System/StorageSystemTables.cpp

```diff
@@ -813,6 +814,14 @@ class TablesBlockSource : public ISource
                         res_columns[res_index++]->insert(dependents_tables);
 
                 }
+
+                if (columns_mask[src_index++])
+                {
+                    if (table && table->getInMemoryMetadataPtr()->sql_security_type == SQLSecurityType::DEFINER)
+                        res_columns[res_index++]->insert(*table->getInMemoryMetadataPtr()->definer);
+                    else
+                        res_columns[res_index++]->insertDefault();
+                }
```

Comment:
I'm lacking a bit of context in this area of the code. And it's a bit vague to understand based on the surrounding function.

Can I ask what this is doing? 😅

Reply:
This adds the `definer` field to the system.tables. It can be helpful for debugging purposes.